### PR TITLE
Savings module withdrawals

### PR DESCRIPTION
**Pull request by denalimarsh**
Withdraw from the saving's module deposit object

### CODE REVIEW

File: x/savings/client/cli/tx.go [hunk 1]
```diff
@@ -42,7 +43,7 @@ func getCmdDeposit() *cobra.Command {
 		Use:   "deposit [amount]",
 		Short: "deposit coins to savings",
 		Example: fmt.Sprintf(
-			`%s tx %s savings 10000000bnb --from <key>`, version.AppName, types.ModuleName,
+			`%s tx %s deposit 10000000bnb --from <key>`, version.AppName, types.ModuleName,
```
Comment (karzak) [inline x/savings/client/cli/tx.go:46]: nit: I would add a multi-coins example

```
`%s tx %s deposit 10000000ukava,100000000usdx --from <key>`, version.AppName, types.ModuleName,

**Review by karzak — APPROVED** [review]
LGTM - tested all the cases I could think of and functionality looks good and error messages are clear.

File: x/savings/client/cli/tx.go (hunk 1 above)
Comment (denalimarsh) [reply x/savings/client/cli/tx.go]: Updated in https://github.com/Kava-Labs/kava/pull/1197/commits/bdbb689d23e2a437d192736146a705c479a4ba50

File: x/savings/client/cli/tx.go [hunk 2]
```diff
@@ -62,3 +63,29 @@ func getCmdDeposit() *cobra.Command {
 		},
 	}
 }
+
+func getCmdWithdraw() *cobra.Command {
+	return &cobra.Command{
+		Use:   "withdraw [amount]",
+		Short: "withdraw coins from savings",
+		Example: fmt.Sprintf(
+			`%s tx %s withdraw 10000000bnb --from <key>`, version.AppName, types.ModuleName,
```
[the remark this answers is not in the document]
Comment (denalimarsh) [reply x/savings/client/cli/tx.go]: Updated in https://github.com/Kava-Labs/kava/pull/1197/commits/bdbb689d23e2a437d192736146a705c479a4ba50

**Pull request merged**